documentation/deploy-page.md

```diff
@@ -0,0 +1,10 @@
+# Deploy page
+
+Right now the offline page is a sign that is enabled via toggle, and with mvc offline.
```

Comment:
In what scenarios the offline page is shown? what happens if the online status is restored when the user is seeing the offline page? Maybe you can document other details.

We could consider using Spanish to make it easy.